src/lib/output/renderer.ts

```diff
@@ -282,6 +285,21 @@ export class Renderer extends ChildableComponent<Application, RendererComponent>
         return Path.dirname(require.resolve('typedoc-default-themes'));
     }
 
+    /**
+     * Return the path to the target installed theme.
+     *
+     * @returns The path to the default theme.
+     */
+    static getInstalledTheme(themeName: string): string {
+        if (/[\/\\]/.test(themeName)) {
+            return null;
+        }
+        try {
+            return Path.join(Path.dirname(require.resolve(`typedoc-themes-${themeName}`)), 'theme');
```

Comment:
should this be generalized to allow for themes other than the default two?

Reply:
the thame looke like this https://github.com/bluelovers/typedoc-themes-color